src/main/java/com/spotinst/sdkjava/model/SpotinstOceanGKELaunchSpecRepo.java

```diff
@@ -0,0 +1,79 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.exception.ExceptionHelper;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.gcp.OceanGKELaunchSpec;
+
+public class SpotinstOceanGKELaunchSpecRepo implements ISpotinstOceanGKECreateLaunchSpecRepo {
```

Comment:
Implement get all function.

Reply:
As we discussed , I have implemented getAllStatus instead getAll